kornia/losses/_utils.py

```diff
@@ -0,0 +1,21 @@
+from typing import Optional
```

Comment:
```suggestion
from __future__ import annotations
from typing import Optional
```
This will fix the tests